src/components/Input/QtySelector/QtySelector.js

```diff
@@ -44,6 +46,12 @@ class QtySelector extends Component {
     focused: false
   };
 
+  componentDidUpdate = () => {
+    if (typeof this.props.callBack === "function") {
```

Comment:
I think invoking this `callBack` function every time `componentDidUpdate` fires may be overkill and it could be redundant with the `onChange` prop.

What do you think about using `onIncrement` and `onDecrement` props that are invoked within the `QtySelector's` `increment` and `decrement` methods instead?